css/styles.css

```diff
@@ -820,6 +863,191 @@ nav .closebtn {
   opacity: 0.96;
 }
 
+.projects-section {
+  display: grid;
+  grid-template-rows: repeat(4, 25%);
+  align-items: center;
+  padding: 115px 25px;
+  gap: 285px;
+  width: 100%;
+  height: 100%;
+  justify-items: center;
+}
+
+/* .tonic-card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 25px;
+  gap: 15px;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+} */
+
+/** details popup css styles */
+.modal {
+  display: none; /* Hidden by default */
+  position: fixed; /* Stay in place */
+  z-index: 9999; /* Sit on top */
+  left: 0;
+  top: 0;
+  width: 100%; /* Full width */
+  height: 100%; /* Full height */
+
+  /* overflow: auto; Enable scroll if needed */
+  background-color: rgb(0, 0, 0); /* Fallback color */
+  background-color: rgba(0, 0, 0, 0.4); /* Black w/ opacity */
+}
+
+/* Modal Content/Box */
+.modal-content {
+  display: inline-block;
+  position: fixed;
+  overflow: auto;
+  left: 50%;
+  top: 1rem;
+  transform: translate(-50%, 0);
+  width: 90%;
+  height: 95vh;
+  border-radius: 16px;
+  padding: 25px;
+  background-color: #fff;
+}

```

Comment:
- [ ] `required`
- [ ] Good job so far 👏 👏 
- [ ] However i would suggest you make changes in accordance with the `figma template` 
- [ ] Please make sure `all` the popups follow the design for both desktop and mobile versions from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g., `font`, `colours`, `images`, `text`, `margins`) see screenshot for comparisons

`figma template`
![figma pop-up](https://user-images.githubusercontent.com/66230306/181854122-2ca4d5da-1f84-482f-96d4-de60edcb784e.png)

`your desktop popup`
![deaktop template](https://user-images.githubusercontent.com/66230306/181854229-8821f32e-2a7e-4c47-9c8f-3fb4cbf116d3.png)